tox.ini

```diff
@@ -29,9 +29,9 @@ commands =
 
     # Override pipenv version of Django as needed by test envs. Use pip so that
     # Pipfile.lock doesn't end up changed by this process.
-    django18: pip install Django~=1.8.0
     django19: pip install Django~=1.9.0
     django110: pip install Django~=1.10.0
+    django111: pip install Django~=1.11.0 
```

Comment:
Do we specifically want this here at this point?

Reply:
It is not necessary here, but I didn't remove it after adding it 'cos it has to be checked later anyway and the tests are passing too.

Reply:
When I run `tox` locally there is a 1.11 failure. I assume that is also why you removed it from .travis.yml after adding it there.

Reply:
The tests passed for me for django 1.11. Not sure why it is failing for you in tox. But yes, I saw it failing in travis.yml. I can get rid of it now and add it later when necessary.

Reply:
ok. For reference, it is failing for me in `tox` same as how it is failing in [Travis](https://travis-ci.org/unt-libraries/django-premis-event-service/jobs/574065567#L285). Not a failure you need to address now, since we aren't trying to be compatible with 1.11 yet.